Overview.src.html

```diff
@@ -3049,6 +3084,39 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  to `<code title>GET</code>` and <var>request</var>'s <span title=concept-request-body>body</span>
  to null.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <span title=concept-header-parse>parsing</span>
+   `<code title>Include-Referred-Token-Binding-ID</code>` in <var>actualResponse</var>'s
+   <span title=concept-response-header-list>header list</span>.
+   The value of this <span title=concept-header>header</span> is specified in
+   <a href="https://tools.ietf.org/id/draft-ietf-tokbind-https-05.xml#rfc.section.3.3">section 3.3</a>
+   of the Token Binding over HTTP spec.
+   <p class="note no-backref">By setting the <code title>Include-Referred-Token-Binding-ID</code>
+   <span title=concept-header>header</span> to <var>true</var>,
+   the <span title=concept-request-origin>origin</span> that sends the
+   redirect <span title=concept-response>response</span> tells the user agent to disclose the
+   <span title=concept-token-binding-id>Token Binding Id</span> used by the user agent with that
+   <span title=concept-request-origin>origin</span> to the target
+   <span title=concept-request-origin>origin</span>.
+
+ <li><p>Unset the request's <var>use-referred-token-binding</var> flag.
```

Comment:
Here and below you don't use `<var>request</var>` which is probably what you want to be doing. The flag however needs to be referenced using `<span>`, not `<var>`.